tests/tests/tier0/bluechictl-status-watch/python/bluechictl_status_watch.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import re
+import unittest
+
+from bluechi.api import Node
+from bluechi_machine_lib.bluechictl import BackgroundRunner, BluechiCtl, SimpleEventEvaluator
+from dasbus.error import DBusError
+
+node_foo_name = "node-foo"
+service_simple = "simple.service"
+
+
+class TestBluechictlStatusWatch(unittest.TestCase):
+
+    def setUp(self) -> None:
+        node_status_offline = re.compile(r"^" + re.escape(node_foo_name) + ".*offline")
+        node_status_online = re.compile(r"^" + re.escape(node_foo_name) + ".*online")
+        expected_patterns = {node_status_online,
+                             node_status_offline}
+        args = ["status", "-w"]
+        self.evaluator = SimpleEventEvaluator(expected_patterns)
+        self.bgrunner = BackgroundRunner(args, self.evaluator)
+        self.bluechictl = BluechiCtl()
```

Comment:
`self.bluechictl` isn't used and can be removed.

Reply:
Done, thanks